workstation/w_linux.go

```diff
@@ -96,10 +96,10 @@ func (l *linux) ListRemovableDisk() error {
 	}
 
 	if !(len(out) > 0) {
-		return fmt.Errorf("[-] No mounts found.\n[-] Please insert your SD card and start command again\n")
+		return nil, fmt.Errorf("[-] No mounts found.\n[-] Please insert your SD card and start command again\n")
```

Comment:
__[golint]__
* [golint] error strings should not be capitalized or end with punctuation or a newline

<!-- Comment by [SideCI](https://sideci.com) -->